gateway/1.8/modules/ROOT/pages/_partials/task-config-readiness-liveness.adoc

```diff
@@ -127,6 +130,28 @@ spec:
    enabled: true
    port: 3000
 ----
++
+Applying the YAML configuration snippet enables the readiness API to respond to `GET` requests:
++
+`\http://<flex-gateway-url>:<port-configured>/probes/readiness?allowEnvoyErrors=<true|false>&allowAPIErrors=<true|false>&allowPolicyErrors=<true|false>`
++
+[discrete]
+=== Configure Error Tolerance
++
+You can configure the readiness probe to be error-intolerant, meaning that misconfigured gateway instances never reach a readiness state of `true`.
++
+The `flexctl probe` accepts the following flags to configure error tolerance:
+
++
+* `--allow-api-errors`
```

Comment:
The syntax of these flags are for the k8s helm chart. Plus, I don't think they should be called "flags", since we're really talking about query params here.

Maybe these don't even need to be included since you're already mentioning the query params below.